examples/02_decoding/plot_haxby_decoder.py

```diff
@@ -0,0 +1,58 @@
+"""
+Decoding with decoder metaestimator: face vs house recognition
+==============================================================
+
+Here is a simple example of decoding with a the decoder metaestimator,
+reproducing a face vs house discrimination task on the study "Distributed and
+Overlapping Representations of Faces and Objects in Ventral Temporal Cortex"
+(Science 2001).
+
+"""
+
+#########################################################################
+# Load Haxby dataset
+import numpy as np
+
+# Load the Haxby dataset
+from nilearn.datasets import fetch_haxby
+data_files = fetch_haxby(n_subjects=1)
+
+func_filenames = data_files.func[0]
+labels_filenames = data_files.session_target[0]
+
+from nilearn.image import mean_img
+background_img = mean_img(func_filenames)
+
+# Load Target labels
+labels = np.recfromcsv(labels_filenames, delimiter=" ")
+
+# Restrict to face and house conditions
+target = labels['labels']
+condition_mask = np.logical_or(target == "face", target == "house")
+
+# Split data into train and test samples, using the chunks
+condition_mask_train = np.logical_and(condition_mask, labels['chunks'] <= 6)
+condition_mask_test = np.logical_and(condition_mask, labels['chunks'] > 6)
+
+### Prediction with Decoder ###################################################
```

Comment:
Can you change this line to same as what you have done above for Load Haxby dataset.
starts with ##### and then your headline or title.